tests/profiling/exporter/test_packages.py

```diff
@@ -0,0 +1,67 @@
+import mock
+import pytest
+
+from ddtrace.profiling.exporter import _packages
+from ddtrace.profiling.exporter._packages import pathlib
+
+
+@pytest.mark.parametrize(
+    "filename,result",
+    (
+        ("toto.py", True),
```

Comment:
will we ever get anything from `__pycache__` ?

Reply:
Not from what I saw when testing.